homeassistant/components/fortigate/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""Fortigate integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_USERNAME, CONF_PASSWORD, \
+    CONF_DEVICES, EVENT_HOMEASSISTANT_STOP
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.discovery import async_load_platform
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'fortigate'
+
+DATA_FGT = DOMAIN
+
+ATTR_NAME = 'name'
+DEFAULT_NAME = 'Default State'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_DEVICES, default=[]):
+            vol.All(cv.ensure_list, [cv.string]),
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+
+async def async_setup(hass, config):
+    """Start the fortigate component."""
+    conf = config.get(DOMAIN)
+
+    if conf is not None:
```

Comment:
This case is the only one that can happen since there is no network discovery or config flow.

Reply:
I'm not sure to get the point ?

Reply:
Please remove this line and outdent below.

Reply:
fixed, thanks !